pkg/scte35/segmentation_upid.go

```diff
@@ -168,18 +170,15 @@ func (upid *SegmentationUPID) Name() string {
 // range are represented by a dot (.).
 func (upid *SegmentationUPID) ASCIIValue() string {
 	b := upid.valueBytes()
-	var stringsValues []string
-	for _, b := range b {
-		var stringVal string
-		if utf8.Valid([]byte{b}) {
-			stringVal = string(b)
-		} else {
-			// non valid ascii byte element, it is represented as a dot
-			stringVal = "."
+	rs := make([]byte, 0, len(b))
+	for i := range b {
+		if b[i] <= 127 {
```

Comment:
I think this is supposed to be ASCII printable characters, which means the line should be something like:

    if b[i] > 31 && b[i] < 127